lib/modules/datasource/metadata.spec.ts

```diff
@@ -256,4 +261,180 @@ describe('modules/datasource/metadata', () => {
       'https://example.com/foo/bar'
     );
   });
+
+  it('Should delete homepage if its not a link to a path in github repo', () => {
+    const dep = {
+      homepage: 'https://github.com/foo/bar',
+      releases: [
+        { version: '1.0.1', releaseTimestamp: '2000-01-01T12:34:56' },
+        { version: '1.0.2', releaseTimestamp: '2000-01-02T12:34:56.000Z' },
+        { version: '1.0.3', releaseTimestamp: '2000-01-03T14:34:56.000+02:00' },
+      ],
+    };
+    addMetaData(dep, MavenDatasource.id, 'foobar');
+    expect(dep).toMatchObject({
+      releases: [
+        {
+          version: '1.0.1',
+          releaseTimestamp: '2000-01-01T12:34:56.000Z',
+        },
+        {
+          version: '1.0.2',
+          releaseTimestamp: '2000-01-02T12:34:56.000Z',
+        },
+        {
+          version: '1.0.3',
+          releaseTimestamp: '2000-01-03T12:34:56.000Z',
+        },
+      ],
+      sourceUrl: 'https://github.com/foo/bar',
+    });
+  });
+
+  it('Should delete homepage if its same as sourceUrl after massage', () => {
+    const dep = {
+      homepage: 'http://somesource.com',
+      sourceUrl: 'http://somesource.com',
```

Comment:
I'm not sure, but i think this causes duplicate links in PR with same taget, so one probably `homepage` should removed?

@rarkins WDYT?

Reply:
Yes this tests that (duplicate links).
If both are same then only `sourceUrl` will be kept right?

Reply:
The test name is `Should delete homepage if its same as sourceUrl after massage` 
Can you please explain further where is the problem in the test ?

Reply:
test looks ok now